addons/common/XEH_preInit.sqf

```diff
@@ -215,6 +215,7 @@ PREP(getConfigGunner);
 PREP(getConfigCommander);
 PREP(getHitPoints);
 PREP(getHitPointsWithSelections);
+PREP(getSelectionsWithoutHitPoints);
```

Comment:
>Why not just make them into one function anyways and parameterize it?
It's a completely different approach as getHitpoints. I personally don't like the overhead that comes with checking optinal parameters when nothing but one line is shared between both functions.